envoyauth/request_test.go

```diff
@@ -209,6 +252,9 @@ func TestGetParsedBody(t *testing.T) {
 		"content_type_json_with_special_chars": {input: createCheckRequest(requestBodyWithJSONSpecialChars), want: expectedJSONSpecialChars, isBodyTruncated: false, err: nil},
 		"empty_content":                        {input: createCheckRequest(requestEmptyContent), want: nil, isBodyTruncated: false, err: nil},
 		"body_truncated":                       {input: createCheckRequest(requestBodyTruncated), want: nil, isBodyTruncated: true, err: nil},
+		"content_type_url_encoded":             {input: createCheckRequest(requestContentTypeUrlEncoded), want: expectedUrlEncodedObject, isBodyTruncated: false, err: nil},
+		"content_type_url_encoded_empty":       {input: createCheckRequest(requestContentTypeUrlEncodedEmpty), want: nil, isBodyTruncated: false, err: nil},
+		"content_type_url_encoded_truncated":   {input: createCheckRequest(requestContentTypeUrlEncodedTruncated), want: nil, isBodyTruncated: true, err: nil},
```

Comment:
Let's add one with multiple values for one key, maybe? What do you think?

Reply:
good call, fixed in f76cbd6